lib/main.dart

```diff
@@ -78,9 +84,11 @@ Future _runWithLogs(Function() function, {String prefix = ""}) async {
 }
 
 Future<void> _init(bool bool, {String? via}) async {
-  // Start workers asynchronously. No need to wait for them to start
-  Computer.shared().turnOn(workersCount: 4, verbose: kDebugMode);
-  CryptoUtil.init();
+  if (Platform.isWindows) {
+    registerProtocol('unilinks');
+  }
+  await initCryptoUtil();
```

Comment:
We'd have to `turnOn` the computer else where? Or are we discarding the package altogether?

Reply:
We don't have computer now, we are using runIsolated from sodium as discussed before.